packages/legend-studio/src/models/protocols/pure/v1/V1_PureGraphManager.ts

```diff
@@ -1944,62 +1945,51 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
 
   // --------------------------------------------- V1_Store ---------------------------------------------
 
-  generateStore = flow(function* (
+  buildDatabase = flow(function* (
```

Comment:
Let's use the new `flow` form:

```suggestion
  *buildDatabase( ...
```